http/http-api/src/main/java/com/okta/commons/http/config/HttpClientConfiguration.java

```diff
@@ -17,14 +17,32 @@
 package com.okta.commons.http.config;
 
 import com.okta.commons.http.authc.RequestAuthenticator;
+import com.okta.commons.lang.Strings;
+import com.okta.sdk.impl.config.*;
+import com.okta.sdk.impl.io.ClasspathResource;
+import com.okta.sdk.impl.io.DefaultResourceFactory;
+import com.okta.sdk.impl.io.Resource;
+import com.okta.sdk.impl.io.ResourceFactory;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.LinkedHashMap;
+import java.util.Map;
 
 
 /**
  * This class holds the configuration properties use to construct a {@link com.okta.commons.http.RequestExecutor RequestExecutor}.
  *
  * @since 0.5.0
  */
-public class HttpClientConfiguration {
+public class HttpClientConfiguration implements ClientConfiguration {
+
+    private static final String ENVVARS_TOKEN   = "envvars";
+    private static final String SYSPROPS_TOKEN  = "sysprops";
+    private static final String OKTA_CONFIG_CP  = "com/okta/sdk/config/";
+    private static final String OKTA_YAML       = "okta.yaml";
+    private static final String OKTA_PROPERTIES = "okta.properties";
```

Comment:
This duplicates code from the SDK project, don't duplicate the code, just expose the bits needed and parse the config in the SDK

Reply:
fixed